packages/web/src/schema/playlist-schema.js

```diff
@@ -2,12 +2,12 @@ import { normalize, schema } from "normalizr";
 
 const playlist = new schema.Entity("playlists", {}, { idAttribute: "_id" });
 
-const track = new schema.Entity("tracks", {}, { idAttribute: "_id" });
+const song = new schema.Entity("songs", {}, { idAttribute: "_id" });
 
 const fullPlaylist = new schema.Entity(
```

Comment:
The naming of the schemas here are a bit confusing. `fullPlaylist` seems to be the "playlist" schema, and `playlistListSchema` seems to be the "playlists" schema (notice the plural). But then `playlist` is actually a schema for "playlists" too…

It's a common pattern to have an entity in singular and a collection of entities in plural. If that's what you're trying to represent with those schemas, I'd go for that naming. I would also try to be consistent on how everything is implemented so that the code is ordered and easy to read.

I'd like to have a conversation about this file to better understand it.